### ISSUE TITLE
Remove Annoying Notification?

### ISSUE THREAD

**Issue by dm17** [Telegram-FOSS-Team/Telegram-FOSS#441]
There's a default notification called "Push service: tap to learn more" that always shows the telegram logo in the upper right... Leading one to believe they have Telegram messages. How does one remove this so they only have the Telegram notification symbol when they actually have a message (which is how it should work I believe). Keeping a permanent tutorial as a notification in the menu seems intrusive.

**Comment by stefan-niedermann** [comment]
It is [clearly described](https://github.com/Telegram-FOSS-Team/Telegram-FOSS/blob/master/Notifications.md) why the notification has to be there and how it can be hidden.

> You may still lower the priority of the notification channel or even hide it altogether manually (make a long tap on the notification).

**Comment by dm17** [comment]
> It is [clearly described](https://github.com/Telegram-FOSS-Team/Telegram-FOSS/blob/master/Notifications.md) why the notification has to be there and how it can be hidden.
> 
> > You may still lower the priority of the notification channel or even hide it altogether manually (make a long tap on the notification).

That doesn't describe what I see.
1) Long tap the notification -> shows "Push Notifications Service" (Alerting | Silent) ... "Turn off notifications" | Done
2) There is nothing to long tap (after the first long tap)

Sorry, but what am I missing here? Thanks.

**Comment by stefan-niedermann** [comment]
Tapping on "Turn off notifications" should open a list of all notification channels which are related to Telegram. (This is "Android" and nothing special about Telegram. Other apps are using multiple notification channels, to.

I don't know exactly what the channel for the "annoying notification" was called, maybe "Others" or so. On the right side should be a switch control to disable this notification channel.

You *might* get another notification for "heavy" battery usage then, which you can disable the same way.

I am sure that a Pull Request with enhanced wording, maybe even a few screenshots will be welcome to the maintainers of this repository.

**Comment by dm17** [comment]
Thanks! Much better... It was "Push Notifications Service" -> off
I was thinking this was turning off the push notification service for Telegram (which is understandable considering the language)!

**Comment by mohzart** [comment]
I had the same problem, and since I'm a little OCD'ish (especially with my phone) ;) 

...i would suggest to underline this issue either in the nfo, or (imo BOTH) rename:

Push Notificaton Service 
TO ...idk ...really(!)
FOSS Info Notification
...or even :P
"Annoing FOSS Notification"

Because not everyone goes through nfo line by line ;)

**Comment by realityexists** [comment]
I agree, the name is confusing. When I go to notifications options for the app and disable "Push Notification Service" it sounds like I'm disabling all push notifications for the app. Rename it to something obvious like "Info only - it's safe disable this notification".
